x-pack/plugins/case/server/types.ts

```diff
@@ -4,19 +4,27 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-// eslint-disable-next-line @kbn/eslint/no-restricted-paths
-import { AppRequestContext } from '../../security_solution/server/types';
+import type { IRouter, RequestHandlerContext } from 'src/core/server';
+import type { AppRequestContext } from '../../security_solution/server';
+import type { ActionsApiRequestHandlerContext } from '../../actions/server';
 import { CaseClient } from './client';
 
 export interface CaseRequestContext {
   getCaseClient: () => CaseClient;
 }
 
-declare module 'src/core/server' {
-  interface RequestHandlerContext {
-    case?: CaseRequestContext;
-    // TODO: Remove when triggers_ui do not import case's types.
-    // PR https://github.com/elastic/kibana/pull/84587.
-    securitySolution?: AppRequestContext;
-  }
+/**
+ * @internal
+ */
+export interface CasesRequestHandlerContext extends RequestHandlerContext {
+  case: CaseRequestContext;
+  actions: ActionsApiRequestHandlerContext;
+  // TODO: Remove when triggers_ui do not import case's types.
+  // PR https://github.com/elastic/kibana/pull/84587.
```

Comment:
Super nit -> The comments should be removed right? To my understanding it is required to declare all plugin dependencies context and we should not move the `securitySolution`, right?

Reply:
@cnasikas no idea, this comment was copied from the old declaration https://github.com/elastic/kibana/pull/88718/files/6334f55c5e4a8af3cc93d10345cb2340e14fc437#diff-e0b5bc10c412de3c7f41b1647bfefd4c32bf449eb235c1b257e8eac0daf6da80L18